service/controller/aws/v21/version_bundle.go

```diff
@@ -52,6 +52,16 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Updated to v0.4.1. https://github.com/giantswarm/metrics-server-app/blob/db61c5c13c4ba55c357ab098253b3f3fe8f4e2cd/CHANGELOG.md#v041",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "cert-exporter",
+				Description: "Migrated to use default app catalog.",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "net-exporter",
+				Description: "Migrated to use default app catalog.",
+				Kind:        versionbundle.KindChanged,
```

Comment:
I think the changelogs are more readable if we have all the migrated apps together in alpha order.

I don't think the changelogs need to be sequential. As they only apply when the release is activated. WDYT?

Reply:
Adjusted, PTAL